.gitlab-ci.yml

```diff
@@ -6,7 +6,7 @@ variables:
   VERSION: dev
   # Manual trigger variables
   TRACER_BRANCH:
-    description: "Branch of the dd-trace-dotnet repository to use (default empty)."
+    description: "Branch of the dd-trace-dotnet repository to use (default empty). For a basic testing, you can use v3.2.1 or something, the latest released version with a v in front of it."
```

Comment:
This is wrong, its literally branches, it just happens that version branches are `vX.X.X`

Reply:
Maybe recommend `master`